WikiWatershed/gwlf-e
# Upgrade to Python 3.9

## Overview

Upgrades codebase to Python 3.9, drops support for Python 2.7. See commit messages for details.

Connects #86 
Connects https://github.com/WikiWatershed/model-my-watershed/issues/3165

## Testing Instructions

* Ensure you have Python 3.9 and [pipenv](https://pipenv.pypa.io/en/latest/) installed
* Checkout this branch
* Run `pipenv sync`
* Run `pipenv run nosetests`
